compiler/config.py

```diff
@@ -15,13 +15,16 @@
 else:
     PASH_TOP = subprocess.run(GIT_TOP_CMD, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True).stdout.rstrip()
 
+PARSER_BINARY = os.path.join(PASH_TOP, "compiler/parser/parse_to_json.native")
+PRINTER_BINARY = os.path.join(PASH_TOP, "compiler/parser/json_to_shell.native")
```

Comment:
Similarly, are those needed? We are now using Thurston's infrastructure so parsing and printing should not happen through ocaml